tests/MakersOfDenmark.WebAPI.Tests/RegisterMakerSpaceHandlerTest.cs

```diff
@@ -0,0 +1,38 @@
+﻿using AutoFixture;
+using FluentAssertions;
+using MakersOfDenmark.Application.Commands.V1;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace MakersOfDenmark.WebAPI.Tests
+{
+    public class RegisterMakerSpaceHandlerTest : IClassFixture<RequestHandlerFixture>
+    {
+        private readonly RequestHandlerFixture _requestHandlerFixture;
+
+        public RegisterMakerSpaceHandlerTest(RequestHandlerFixture requestHandlerFixture)
+        {
+            _requestHandlerFixture = requestHandlerFixture;
+        }
+        [Fact]
+        public async Task RegisterMakerSpaceTest()
+        {
+            var request = _requestHandlerFixture.Fixture.Build<RegisterMakerSpace>()
+                .With(x => x.LogoUrl, "https://localhost")
+                .With(x => x.AccessType, "Public")
+                .Create();
+
+            var handler = new RegisterMakerSpaceHandler(_requestHandlerFixture.DbContext);
+
+            _requestHandlerFixture.DbContext.MakerSpace.Should().HaveCount(0);
+            var creater = await handler.Handle(request);
```

Comment:
Eventuelt et assert på at Id bliver sat? Måske at den ikke returnere default værdien for en Guid?